ecs-cli/modules/utils/utils.go

```diff
@@ -106,3 +107,45 @@ func GetPartition(region string) string {
 		return "aws"
 	}
 }
+
+// ParseLoadBalancers parses a StringSlice array into an array of load balancers struct
+// Input: ["targetGroupArn="...",containerName="...",containerPort=80","targetGroupArn="...",containerName="...",containerPort=40"]
+func ParseLoadBalancers(flagValues []string) ([]*ecs.LoadBalancer, error) {
+	var list []*ecs.LoadBalancer
+
+	for _, flagValue := range flagValues {
+		m := make(map[string]string)
+		elbv2 := false
+		keyValPairs := strings.Split(flagValue, ",")
+
+		for _, kv := range keyValPairs {
+			pair := strings.SplitN(kv, "=", -1)
+			if len(pair) > 2 {
+				return nil, fmt.Errorf("Only include one = to indicate your value in your %s", pair[0])
+			}
+
+			m[pair[0]] = pair[1]
+			if pair[0] == "targetGroupArn" {
+				elbv2 = true
+			}
+		}
+		containerPort, err := strconv.ParseInt(m["containerPort"], 10, 64)
+		if err != nil {
+			return nil, fmt.Errorf("Fail to parse container port %s for container %s", m["containerPort"], m["containerName"])
+		}
+		if elbv2 {
+			list = append(list, &ecs.LoadBalancer{
```

Comment:
Might also need to do the validation as well to make sure
1) at least one of `targetGroupArn` and `loadBalancerName` exists
2) containerName and containerPort must exist